Source/WebCore/css/process-css-properties.py

```diff
@@ -2575,17 +2575,21 @@ def _generate_lookup_functions(self, *, to):
                 if (!nameForCSS)
                     return emptyString();
 
+                bool sawHyphen = false;
                 auto* propertyNamePointer = nameForCSS;
                 auto* nextCharacter = characters;
                 while (char character = *propertyNamePointer++) {
                     if (character == '-') {
+                        sawHyphen = true;
                         char nextCharacter = *propertyNamePointer++;
                         if (!nextCharacter)
                             break;
                         character = (propertyNamePointer - 2 != nameForCSS) ? toASCIIUpper(nextCharacter) : nextCharacter;
                     }
                     *nextCharacter++ = character;
                 }
+                if (!sawHyphen)
+                    return nameString(id);
```

Comment:
These changes look unrelated

Reply:
Oh yes, they are. It’s performance optimization that I included intentionally, forgot to mention in the change log, and am happy to land separately.

Reply:
seems fine